src/components/AddBlogPanel/index.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import './style.css';
+
+class AddBlogPanel extends Component {
+    constructor() {
+        super();
+        this.state = {
+            fields: {},
+            errors: {}
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+        this.submitaddBlogForm = this.submitaddBlogForm.bind(this);
```

Comment:
In ES6 exists arrow function which gives a possibility avoid a bind(this). But it's not a necessary.